app/controllers/admins_controller.rb

```diff
@@ -51,7 +51,7 @@ def weekly_user_stats
   end
 
   def stats
-    @popular_tags = ActsAsTaggableOn::Tagging.joins(:tag).limit(50).order('count(taggings.id) DESC').group(:tag).count
+    @popular_tags = ActsAsTaggableOn::Tagging.limit(50).order('count(taggings.id) DESC').group(:tag).count
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.